integreat_cms/deepl_api/utils.py

```diff
@@ -147,12 +183,11 @@ def deepl_translation(self, request, content_objects, language_slug, form_class)
                 existing_target_translation = content_object.get_translation(
                     target_language.slug
                 )
-                # For some languages, the DeepL client expects the BCP tag instead of the short language code
-                if target_language.slug in ("en", "pt"):
-                    target_language_key = target_language.bcp47_tag
-                else:
-                    target_language_key = target_language.slug
-
+                target_language_key = self.get_target_language_key(
+                    request, target_language
+                )
+                if target_language_key is None:
+                    break
```

Comment:
```suggestion
                if not target_language_key:
                    # re-add error message here
                    return
```

I would also move this up and out of the loop, right below `target_language = ...`